dev/CommonStyles/PasswordBox_themeresources.xaml

```diff
@@ -277,7 +278,8 @@
                             Grid.Column="1"
                             Style="{StaticResource RevealButtonStyle}"
                             BorderThickness="{TemplateBinding BorderThickness}"
-                            Margin="{ThemeResource HelperButtonThemePadding}"
+                            contract7Present:CornerRadius="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=CornerRadius, Converter={StaticResource RightCornerRadiusFilterConverter}}"
+                            Padding="{ThemeResource HelperButtonThemePadding}"
```

Comment:
Why changing from Margin to Padding?

Border thickness used to be 2, that's probably why there was a -2 right margin. We changed thickness to 1 now so the margin needs to be updated to -1 as well, then you shouldn't need to change Margin to Padding I think.

Reply:
`ButtonLayoutGrid` (root node) and `RevealButton` have the same BorderThickness, so we should not use a margin on this button to perfectly align their borders (including the rounded corners).

For `HelperButtonButtonButtonThemePadding`, it is possible that this negative padding/margin has been chosen to counterbalance the border, so if it's the case, I agree, it should be `0, 0, -1, -, 0`, but I don't think this change should be part of this PR.